eq-author-api/schema/typeDefs.js

```diff
@@ -138,27 +139,14 @@ type MultipleChoiceAnswer implements Answer {
   guidance: String
   qCode: String
   label: String
+  secondaryLabel: String
   type: AnswerType!
   options: [Option]
   mutuallyExclusiveOption: Option
   page: QuestionPage
   properties: JSON
 }
 
-type CompositeAnswer implements Answer {
```

Comment:
If this doesn't exist can we get rid of `BasicAnswer` and just have `Answer`?

Reply:
Attempting to keep changes to a minimum for the moment. Also I'm pretty sure we would get a naming conflict with `Answer` interface.

Reply:
Can we just remove that interface?

Reply:
It results in a lot of changes on the UI (which we're trying to leave as untouched as possible).